tests/unit/api/objectCopy.js

```diff
@@ -338,6 +339,22 @@ describe('non-versioned objectCopy', () => {
             },
         ], done);
     });
+
+    it('should fail to copy object when setting a crr location as the locationConstraint', done => {
+        const testObjectCopyRequest = _createObjectCopyRequest(destBucketName, {
+            'x-amz-metadata-directive': 'REPLACE', // needed to take the locationConstraint into account
+            [objectLocationConstraintHeader]: 'location-crr-v1',
+        });
+
+        async.series([
+            next => objectPut(authInfo, testPutDestObjectRequest, undefined, log, next),
+            next => objectCopy(authInfo, testObjectCopyRequest, sourceBucketName, objectKey,
+                undefined, log, next),
+        ], err => {
+            assert(err.is.InvalidArgument);
```

Comment:
is this the right error?
should we not sent `InvalidLocationConstraint` in that case (i.e. trying to write on a valid location, but which does not support writing) instead of `InvalidArgument` (simply because we have no backend class for the type `crr` I guess?)

Reply:
Looking at this test, it seems like there was a reason behind returning this error https://github.com/scality/cloudserver/blob/96a0315a8b7d436e4945630b4c0821bf5e7480aa/tests/unit/multipleBackend/locationConstraintCheck.js#L37-L46